src/msg.hpp

```diff
@@ -221,11 +251,11 @@ class msg_t
             metadata_t *metadata;
             unsigned char
               unused[msg_t_size
-                     - (sizeof (metadata_t *) + 2 + 16 + sizeof (uint32_t))];
+                     - (sizeof (metadata_t *) + 2 + sizeof (uint32_t) + 16)];
```

Comment:
sizeof(struct groupt_t) instead of 16, since it's now a struct? Would be more readable

Reply:
fixed